langchain4j-core/src/main/java/dev/langchain4j/model/chat/request/ToolChoice.java

```
@@ -15,5 +15,10 @@ public enum ToolChoice {
     /**
      * The chat model is required to use one or more tools.
      */
-    REQUIRED
+    REQUIRED,
+
+    /**
+     * The chat model is force use no one tool
+     */
+    NONE,
```

Please update `if (toolChoice == REQUIRED)` in `ChatRequestValidationUtils.validate(ToolChoice)` to `if (toolChoice != null && toolChoice != AUTO)` to make sure if someone will specify `ToolChoice.NONE` with the provider that does not support it, it should fail fast to avoid confusion

---

Please update the error message as well:

```java
    public static void validate(ToolChoice toolChoice) {
        if (toolChoice != null && toolChoice != AUTO) {
            throw new UnsupportedFeatureException(String.format("%s.%s is not supported yet by this model provider",
                    ToolChoice.class.getSimpleName(), toolChoice));
        }
    }
```